### PR TITLE
Rename `whitelist.txt` to `allowlist.txt`

### PR DESCRIPTION
**Pull request by zloyden**
Related to https://github.com/AdguardTeam/AdguardFilters/issues/127906.
Do not merge this for now.

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: filters/filter_11_Mobile/template.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@
 !---------------------------------------------------------------------------!
 !------ White list. Fixing filtration errors (false-positive)---------------!
```
Comment (Sergey-Lyapin) [inline filters/filter_11_Mobile/template.txt:33]: I suppose comments must be changed too.

Comment (Alex-302) [reply filters/filter_11_Mobile/template.txt:33]: Yep. Just remove `White list. ` part

Comment (zloyden) [reply filters/filter_11_Mobile/template.txt]: Done

**Review by Sergey-Lyapin — APPROVED** [review]

File: filters/filter_17_TrackParam/template.txt [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 !---------------------------------------------------------------------------!
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/TrackParamFilter/sections/general_url.txt"
 !---------------------------------------------------------------------------!
-!-------------- Specific web sites ---------------------------------------------!
+!-------------- Specific web sites -----------------------------------------!
 !---------------------------------------------------------------------------!
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/TrackParamFilter/sections/specific.txt"
 !---------------------------------------------------------------------------!
-!-------------- Whitelist ------------------------------------------------------!
+!-------------- Whitelist --------------------------------------------------!
```
Comment (AdamWr) [inline filters/filter_17_TrackParam/template.txt:13]: It looks like that there is one more to change.

Comment (zloyden) [reply filters/filter_17_TrackParam/template.txt:13]: Yep, I saw that. I'm doing this now.

**Review by AdamWr — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**